### ISSUE TITLE
Item filter not visible below 20 line items in the stock entry

### ISSUE THREAD

**Issue by Anoop2312** [frappe/erpnext#37568]
Hi Team,

Facing an issue while filtering the items in the stock entry for below 20 line items.

Steps to replicate:

1. Create new Stock entry.

2. In items table dd multiple items

3. filters in item table is not visible when we add below 20 line items.

Expectation:

1. Need filters to be visible above 10 line items on item table.

Observation:

1. The filter is visible only after adding more than 20 line items, the filter is not visible for below 20 line items.

I am attaching the links below for reference. 

 In stock entry which has item less then 20
![1](https://github.com/frappe/erpnext/assets/148338951/67c98cd9-f873-4cc8-a619-d072507afe15)

 In stock entry which has item more then 20
![2](https://github.com/frappe/erpnext/assets/148338951/fee19cd0-d2ba-4652-b14c-dd385f01fc45)

Thanks and Regards,

Anoop verma

**Comment by moaazmf90** [comment]
I don't believe it's a bug, this is happening in other screens also where you can add items, like sales invoice, quotation...etc
I see it reasonable to have it only in more than 20.
